### PR TITLE
Youtubeapicall

### PR DESCRIPTION
**Pull request by DebolaA**
App can now call youtube comment thread api to pull all comments for the given video id

### CODE REVIEW

**Comment by darkmooink** [comment]
good catch on the dotenv being in the dev dependancies, not the dependencies
Ive noticed that you have removed the template .env file and the ability to have multiple environments. Why?
Regarding page size, it might be better to either not have a default and not pass it if its undefined or to set it to 20. This would prevent bug hunting later when someone makes a request without a size, expecting the YouTube api default of 20 but only getting 1.
I would also suggest putting .prettierrc in the .gitignore file and potentially in your global .gitignore as that is a system level file.
I very much approve on the default video id. Although I was potentially expecting dQw4w9WgXcQ or jNQXAC9IVRw but that might show ive been on the internet too long.

File: src/server.ts [hunk 1]
```diff
@@ -12,6 +12,6 @@ app.listen(PORT, () => {
 
   // Seed the database with some data
   if (environment === "dev") {
-    populateDummyData();
+    // populateDummyData();
```
Comment (chaecramb) [inline src/server.ts:15]: Now that my db pull request has been merged, we have a populateDummyData function, so this doess't need to be commented out.

**Comment by chaecramb** [comment]
> good catch on the dotenv being in the dev dependancies, not the dependencies

The way you'd set it up, with multiple environments, wouldn't we want it in the regular dependencies? The `config.ts` file would still load on a production server and look for a `.env.prod` file.

**Comment by DebolaA** [comment]
> good catch on the dotenv being in the dev dependancies, not the dependencies Ive noticed that you have removed the template .env file and the ability to have multiple environments. Why? Regarding page size, it might be better to either not have a default and not pass it if its undefined or to set it to 20. This would prevent bug hunting later when someone makes a request without a size, expecting the YouTube api default of 20 but only getting 1. I would also suggest putting .prettierrc in the .gitignore file and potentially in your global .gitignore as that is a system level file. I very much approve on the default video id. Although I was potentially expecting dQw4w9WgXcQ or jNQXAC9IVRw but that might show ive been on the internet too long.

Hello Topher,

Thank you for your observations

1. I have introduced .prettierrc so that everyone's code will be formatted same way and there will be no merge conflict therefore it should be in our code base not in .gitignore
2. I used 1 so that it does not affect my youtube api calls. that can be easily changed as we have not decided if we want to implement paging or not - something to be discussed in our stand up today
3. Our application should be able to take any videoId therefore the hardcodded videoId is only for testing purpose - how to introduce dynamic videoId is also meant to be a point of discussion today
4. the env setting that was there gave me issues therefore i had to change to what I have now. once merged, you can try to update it to what you had but please ensure it works fine before pushing.

File: src/server.ts (hunk 1 above)
Comment (DebolaA) [reply src/server.ts:15]: please can we all ensure whatever code that is pushed works and code lines should not cause issue once the next developer pulls

**Comment by DebolaA** [comment]
@darkmooink and @chaecramb dotenv now moved to dev dependencies

**Pull request merged**